topics/ecology/tutorials/lotus2-identifying-fungi/tutorial.md

```diff
@@ -0,0 +1,454 @@
+---
+layout: tutorial_hands_on
+
+title: Identifying Mycorrhizal Fungi from ITS2 sequencing using LotuS2
+
+zenodo_link: https://zenodo.org/records/13710591
+
+abbreviations:
+  SPUN: Society for the Protection of Underground Networks
+  MF: Mycorrhizal fungi
+  EcMF: ectomycorrhizal fungi
+  AMF: arbuscular mycorrhizal fungi
+  eDNA: environmental DNA
+  ITS2: Internal Transcribed Spacer 2
+  SSU: Small Subunit (RNA)
+  OTU: Operating Taxonomic Unit
+  ASV: Amplicon Sequence Variant
+  sdm: simple demultiplexer
+  TSV: tab-separated-values
+
+
+questions:
+- What is the fungal community composition in a given soil sample?
+
+objectives:
+- Understand the files needed for running LotuS2
+- Learn to use datasets from Zenodo in a Galaxy tool
+- Learn to upload data files to Galaxy
+- Learn to run the Galaxy LotuS2 tool and what the parameters mean
+- Understand the output files from LotuS2
+- Understand the structure of the mapping file needed by LotuS2 to link sample metadata to a pair of fastq files
+
+time_estimation: 3H
+
+key_points:
+- LotuS is a metagenomics tool for identifying species and {OTU}s and {ASV}s from sequencing data
+- Galaxy is an easy way to run LotuS2 in the cloud for bioinformatics beginners
+
+contributors:
+- sujaikumar
+- bethanmanley
```

Comment:
If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.